lib/block-patterns/large-header-text-button.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Large header with text and a button.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'       => _x( 'Large header with text and a button.', 'Block pattern title', 'default' ),
+	'categories'  => array( 'header' ),
+	'content'     => '<!-- wp:cover {"url":"https://s.w.org/images/core/5.8/art-01.jpg","id":613,"hasParallax":true,"dimRatio":40,"customOverlayColor":"#000000","minHeight":100,"minHeightUnit":"vh","contentPosition":"center center","align":"full"} -->
```

Comment:
The parallax effect in the pattern preview causes the image to appear cutoff:

<img width="334" alt="Screen Shot 2021-04-13 at 2 28 01 PM" src="https://user-images.githubusercontent.com/5375500/114604192-7ab48d80-9c4d-11eb-8390-603057f4a396.png">

Reply:
I've opened an issue to track this here: https://github.com/WordPress/gutenberg/issues/30814